Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1302,6 +1319,10 @@ extension BrowserViewController: URLBarDelegate {
         let actionMenuPresenter: (URL, Tab, UIView, UIPopoverArrowDirection) -> Void  = { (url, tab, view, _) in
             self.presentActivityViewController(url, tab: tab, sourceView: view, sourceRect: view.bounds, arrowDirection: .up)
         }
+
+        let sendToDeviceAction = {
```

Comment:
I think can actually move the entire `presentSendToDevice` into PhotonActionSheetProtocol. You dont need to create and pass this closure in.